PyTrinamic/connections/tmcl_interface.py

```diff
@@ -245,6 +245,10 @@ def moveBy(self, motor, distance, moduleID=None):
         """
         return self.move(1, motor, distance, moduleID).value
 
+    def referenceSearch(self, commandType, motor, moduleID=None):
+        """Use the TMCL RFS command to search for the reference points."""
+        return self.send(TMCL_Command.RFS, commandType, motor, 0, moduleID).value
+

```

Comment:
This is the most essential addon, because it is needed in order to access the RFS command.